tetris.js

```diff
@@ -229,49 +242,63 @@ module.exports = {
 		return newArr;
 	},
 	
-	buildTetromino: function(stringOfCoordinates){
-		let math = stringOfCoordinates.split("\n")
-		let arrayOfCoordinates = []
+	getCoordinates: function(stringOfCoordinates){
+		let fieldArray
 		let quantityElem = 0;
+		let arrayOfCoordinates = []
+		if(typeof(stringOfCoordinates) == "string"){
+			fieldArray = stringOfCoordinates.split("\n")
+		}else{
+			fieldArray = stringOfCoordinates
+		}
 		
-		for(let i = 0, elem_Y = 0; i < math.length; i++){
-			if(math[i]!= false){
+		for(let i = 0, axisElement_Y = 0; i < fieldArray.length; i++){
+			if(fieldArray[i]!= false){
 				
-				for(let j = 0, elem_X = 0; j < math[i].length; j++){
+				for(let j = 0, axisElement_X = 0; j < fieldArray[i].length; j++){
 					let obj = {}
-					if(math[i][j]!= false){
-						if(math[i][j] == "X"){
-							console.log(math[i][j])
-							obj.x = elem_X
-							obj.y = elem_Y
+					if(fieldArray[i][j]!= false){
```

Comment:
DOn't compare strings with booleans, please

Reply:
why can't i use booleans?
Everything works, I do not see the error

Reply:
https://stackoverflow.com/a/359509/362721

Reply:
//typeof(fieldArray[i][j] ==="string"
I checked what kind of object it is "string"
if(fieldArray[i][j]!== "string"){

Reply:
Question: will you ever encounter situation when fieldArray[i][j] is not of type string?